packages/copy-widget-code/src/utils/widgetCode.js

```diff
@@ -0,0 +1,22 @@
+const getBaseUrl = domain => {
+  if (domain === 'localhost') {
+    return 'http://localhost:3000'
```

Comment:
could also be localhost:8080 if the nice backend is run locally and localhost:8080/tocco is opened in browser (maybe that's even more often the case than localhost:3000).